check-sql.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Verify the the working tree modifices no sql files relative to the main
+# branch. This will always pass on a deploy because the working tree is an
+# unmodified copy of the main branch.
+
+target_treeish=${HAIL_TARGET_SHA:-$(git merge-base main HEAD)}
+
+modified_sql_file_list=$(mktemp)
+
+git diff --name-status $target_treeish sql \
+    | grep -Ev '^A|^M\t[^/]+/sql/estimated-current.txt' \
+           > $modified_sql_file_list
+
+if [ "$(cat $modified_sql_file_list | wc -l)" -ne 0 ]
```

Comment:
This didn't work for me locally. Might still be right?

```
wmecc-475:hail jigold$ $(cat $modified_sql_file_list | wc -l)
-bash: 2: command not found
wmecc-475:hail jigold$ cat $modified_sql_file_list | wc -l
       2
```

Reply:
You're getting a command not found because bash resolves
```
 $(cat $modified_sql_file_list | wc -l)
```
To the string
```
        2
```
Which it then tries to execute, but there is no program named "2".